tests/unit/s2n_tls13_handshake_test.c

```diff
@@ -394,6 +394,7 @@ int main(int argc, char **argv)
         EXPECT_NOT_NULL(client_conn = s2n_connection_new(S2N_CLIENT));
         EXPECT_NOT_NULL(server_conn = s2n_connection_new(S2N_SERVER));
         EXPECT_SUCCESS(s2n_connection_set_config(server_conn, server_config));
+        EXPECT_SUCCESS(s2n_connection_set_config(client_conn, s2n_fetch_unsafe_client_testing_config()));
```

Comment:
ideally https://github.com/awslabs/s2n/issues/1477 should be fixed so this line can be removed.

Reply:
You're right, the cert does currently validate without this enabled, but we want to ensure that these tests run regardless of the age of our certificates and that the output on certificate validation failure is clear. If someone is building with a specific version of S2N for a number of years, the cert will eventually expire and they'll have to update, which could cause problems. Ideally our test certificates would be set to expire in a hundred years, which is true for most but not all, so I've opened issue https://github.com/awslabs/s2n/issues/1557 to address that. We also do want our certificate validation tests to fail if a test certificate is expired, but having it fail in other tests too (like the handshake test) would be more confusing than helpful. It's much clearer to the user that the problem is with the certificate if only the certificate validation fails.